storage/transfer/create.go

```diff
@@ -5,13 +5,31 @@ import (
 	domainTransfer "github.com/Erikaa81/banco-erika/domain/transfer"
 )
 
-func (r Repository) Create(transfer domainTransfer.CreateInput) error {
-	r.storage[transfer.Transfer.ID] = entities.Transfer{
-		ID:                   transfer.Transfer.ID,
-		AccountOriginID:      transfer.OriginAccount.ID,
-		AccountDestinationID: transfer.DestinationAccount.ID,
-		Amount:               transfer.Transfer.Amount,
-		CreatedAt:            transfer.Transfer.CreatedAt,
+func (r Repository) Create(input domainTransfer.CreateInput) error {
+	r.storage[input.Transfer.ID] = entities.Transfer{
+		ID:                   input.Transfer.ID,
+		AccountOriginID:      input.OriginAccount.ID,
+		AccountDestinationID: input.DestinationAccount.ID,
+		Amount:               input.Transfer.Amount,
+		CreatedAt:            input.Transfer.CreatedAt,
+	}
+
+	r.storageAccount[input.OriginAccount.ID] = entities.Account{
+		ID:        input.OriginAccount.ID,
+		Name:      input.OriginAccount.Name,
+		CPF:       input.OriginAccount.CPF,
+		PIN:       input.OriginAccount.PIN,
+		Balance:   input.OriginAccount.Balance,
+		CreatedAt: input.OriginAccount.CreatedAt,
+	}
```

Comment:
outra forma de escrever isso seria:

```suggestion
	r.storageAccount[input.OriginAccount.ID] = input.OriginAccount
```